### DISCUSSION TITLE
Solution for finding 8 digit code

### DISCUSSION THREAD

**Discussion by lans4r** [audiamus/AaxAudioConverter#138] [General]
Install the Audiblesync program like usual and download an AAX file.

Drag the file into this site: _Link removed_

💕copy checksum into this very beautiful and wonderful program!

Just tested it works! 😍

**Comment by audiamus** [comment]
Use new [Book Lib Connect](https://github.com/audiamus/BookLibConnect) for downloading and decrypting, as suggested in the readme to this project. This has also the advantage of getting explicit chapter titles, if provided by Audible.

Alternatively, if you desperately want the original AAX format, you can also still go the official way via legacy AudibleManager, which appears to be available on Cnet.

**Reply by Sailfish** [reply]
Is there a field on the Book Lib Connect panel(s) that indicates how many book titles were loaded? I was able to determine it, I believe?, by selecting the first title item on the list and then entering ctl-A to Select All then doing a ctl-C to Copy the list then doing a ctl-V to Paste in my text editor and scrolling to the end and, after subtracting 1 from the line number to account for the heading line.

**Reply by audiamus** [reply]
_Very much off-topic for this thread, but anyway:_ 

No, there are no statistics functions at the moment. But copying the entire table with Ctrl+A/Ctrl+C is pretty smart, I think. I wasn't even aware that the underlying Windows Forms control "DataGridView" supports this. You get a CSV-style output, with a tab as the column separator here instead of the standard comma. But any spreadsheet software will handle the import. The "state" column will remain empty, though, since it's an image column.

**Reply by Sailfish** [reply]
Regrets for commenting incorrectly; I will create a separate discussion in the future.
